install_config/install/advanced_install.adoc

```diff
@@ -2287,7 +2287,24 @@ xref:running-the-advanced-installation-individual-components[Running Individual
 The RPM-based installer uses Ansible installed via RPM packages to run playbooks
 and configuration files available on the local host. To run the installer, use
 the following command, specifying `-i` if your inventory file located somewhere
-other than *_/etc/ansible/hosts_*:
+other than *_/etc/ansible/hosts_*.
+
+
+First, running prerequisites will configure base packages and docker. It needs
```

Comment:
Consider formatting to match modular templates:

Prequisites

Run `prequisites.yml` to configure the base packages and Docker.  This needs to be run only once between deploying a new cluster.

CODE HERE

Procedure

. Install OpenShift:

Reply:
If reformatting is not desired, rewording is still necessary to make the first sentence clearer.

"Run `prequisites.yml` to configure the base packages and Docker.  This needs to be run only once between deploying a new cluster."